src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+All HTML elements except for the ones below will be removed:
+
+.. code-block:: html
+
+    <u> <em> <ul> <li> <p> <strong> <br>
+    
+
+======= =========================================================================================================
+Example .. code-block:: html 
+    
+            <ad:description><![CDATA[
+                <p><strong><u>De goedkoopste webshop</u></strong> 
+                    <strong>voor tweedehands fietsen met garantie! Gratis en rijklaar thuisbezorgd!</strong>
+                </p>
+                <p><strong><br></strong>
+                </p>
+                <ul>
+                    <li><strong>Laagste prijsgarantie</strong></li>
+                    <li>Fietsen <strong>100% rijklaar</strong> gratis thuisbezorgd</li>
+                    <li><strong>Ruime voorraad</strong>, voor ieder wat wils</li>
+                    <li>Snelle <strong>customer service</strong> via Whatsapp, bellen en e-mail</li>
+                    <li>1 <strong>maand garantie</strong></li>
+                    <li>Aangesloten bij <strong>Webwinkelkeur</strong></li>
+                </ul>
+                <strong><br></strong>
+                <p>Check dus snel onze website en vind de fiets die bij je past!<br>
+                </p>
+                <strong><br></strong>
+                <p>WhatsApp, bel of mail ons voor verdere vragen.
+                </p>]]>
+            <ad:description/>
+======= =========================================================================================================
+
+
+.. index:: categoryId
+.. _feed_categoryId:
+
+categoryId
+""""""""""
+
+The category in which the ad is placed in the :ref:`categories` tree.
+
+Each ad belongs to one and only one category and the category of an ad cannot be changed.
+This field can only be set once during the lifecycle of an ad.
+
+An integer value from the category list. Must be an id of a leaf category with a
+non-zero parent id.
+
+======= ====================================================
+Example	.. code-block:: html
+        
+         <ad:categoryId>945</ad:categoryId>
+======= ====================================================
+
+.. index:: status
+.. _feed_status:
+
+status
+""""""
+
+Ad status. One of the following:
+
+====== ====================================================
+Name   Description
+====== ====================================================
+ACTIVE The will be active (as long as there is budget for it), it can be found on the marketplace.
+PAUSED The ad will be paused, effectively not found on the marketplace.
+====== ====================================================
+
+The provided (desired) status may differ from the resulting one, depending on the other conditions.
+For instance, budget may be depleted, or the seller may have too many active ads already in the category.
+
+======= =============================
+Example .. code-block:: html
+        
+         <ad:status>PAUSED</ad:status>
```

Comment:
can we make the example be what we prefer it to be? I think we rather have our ads be ACTIVE than PAUSED

Reply:
But active is the default, this field makes sense only if you want to pause

Reply:
I'm agreeing with @pmongh on this

Reply:
okay :)